financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,183 @@
+
+== Implementing and Deploying the First Project
+
+Now that you have published the other project as an invokable run configuration, you can call it from within another process using the xref:rpa-builder::toolbox-network-rest-call.adoc[*REST Call* Action Step].
+
+The process should take a list of lookup codes provided via a file and use the xref:rpa-builder::toolbox-flow-control-file-controlled-loop.adoc[*File Controlled Loop* Action Step] to iterate over all codes and use them within the request body of the REST call.
+
+xref:rpa-manager::processautomation-deploy.adoc#deploy-a-process-automation[Deploy the project] and schedule it as needed.
+
+
+[[example-of-adapting-the-crpa-file]]
+== Example of Adapting the CRPA File
+
+[NOTE]
+The example in this guide uses real world data to show you how to adapt a CRPA file to a specific AMS.
+
+All values for the Activity Parameters used in RPA Builder are test values that you must overwrite with your own values when creating the run configuration for the process in RPA Manager.
+
+=== Process Model
+
+The process model contains the following three activities:
+
+. Log in to the AMS using credentials and a database name.
+. Open a record in the AMS database using a lookup code.
+. Copy data from various fields on the record pages to a CSV file.
+//Step 3 above should be a CSV file but is a POST REST call in JSON format to http://mainstreet-demo.us-e2.cloudhub.io/epicpolicy. I (Tasha) wrote the instructions as a CSV. What should the text actually state?
+
+image::rpa-setup-process-model.png[RPA Builder process model]
+
+You can add activities by clicking on an element on the model panel and choosing an element from the context menu. For example, the following image shows how to branch off to a second AMS if you are using more than one and not finding the data you need in the first AMS:
+
+image::rpa-setup-add-activity.png[Add activity to RPA Builder process model]
+
+Click on an activity to open the editing workflow.
+
+=== Activity 1: Log in to the AMS
+
+==== Overview of the Activity Workflow
+
+. Decrypt the AMS login credentials that go in through an Activity Parameter.
+. Run the AMS.
+. Log in to the AMS using the decrypted credentials.
+. Select the database using another in-going Activity Parameter.
++
+image::rpa-setup-activity-parameters-wizard.png[RPA Builder Activity Parameter Wizard]
+
+==== Minimum Steps to Adapt in This Workflow
+
+. Change the following in the xref:rpa-builder::toolbox-control-run-program.adoc[Run Program Action Step]:
+.. Replace the values in the *File name*, *Parameters*, and *Window Title* fields with the values from your AMS.
+.. Click the *Test* button to run the application so that you can use it to adapt the following Action Steps:
++
+image::rpa-setup-run-program-wizard.png[RPA Builder Run Program Wizard]
+
+. Use the *Start from scratch* option to capture and test a new pattern that indicates the login page of the AMS is visible onscreen:
```

Comment:
```suggestion
. Use the *Start from scratch* option to capture and test a new pattern that indicates that the login page of the AMS is visible onscreen:
```

Reply:
Use the *Start from scratch* option > Select *Start from scratch* (if this is something to select).